test/server.test.js

```diff
@@ -1,14 +1,347 @@
-const chai = require('chai');
-const chaiHttp = require('chai-http');
-const server = require('../app/server'); // Adjust the path as needed
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const server = require("../app/server");
 
-const should = chai.should();
+chai.should();
 chai.use(chaiHttp);
 
-// TODO: Write tests for the server
+describe("Login", () => {
+  describe("POST /api/login", function () {
+    it("Should return an access token for authentication if login credentials are valid", function (done) {
+      const login = { username: "yellowleopard753", password: "jonjon" };
 
-describe('Brands', () => {});
+      chai
+        .request(server)
+        .post("/api/login")
+        .send(login)
+        .end((err, res) => {
+          res.status.should.equal(200);
+          res.body.should.be.an("object");
+          res.body.should.have.property("token");
+          res.body.token.should.be.a("string");
+          done();
+        });
+    });
 
-describe('Login', () => {});
+    it("Should return an error message for invalid login credentials", function (done) {
```

Comment:
you should test both scenarios of no user name and no password in different tests